lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -286,11 +285,11 @@ def call(params)
                           puts "Client #{client_id} is outside segments_filter #{params.segments}"
                           next
                         end
-                        project = c.project
-                        fail "Client #{client_id} does not have project." unless project
+                        current_project = c.project
+                        fail "Client #{client_id} does not have project." unless current_project
                         working_client_ids << client_id.to_s
-                        puts "Project #{project.pid} of client #{client_id} will receive #{users.count} users"
-                        project.import_users(users,
+                        puts "Project #{current_project.pid} of client #{client_id} will receive #{users.count} users"
+                        current_project.import_users(users,
```

Comment:
Similar code found in synchronize_users.rb:193, synchronize_users.rb:204, synchronize_users.rb:221, synchronize_users.rb:247, synchronize_users.rb:265, synchronize_users.rb:292, synchronize_users.rb:334 (mass = 112)